MiniApp/Classes/MiniApp.swift

```diff
@@ -47,6 +60,9 @@ public class MiniApp: NSObject {
     ///                         to interact with View component of mini app.
     ///         -   Error: Error details if Mini App View creating is failed
     ///   - messageInterface: Protocol implemented by the user that helps to communicate between Mini App and native application
+    @available(*, deprecated,
```

Comment:
Could you also remove the other deprecated `create` function in this PR (`create(appInfo:completionHandler)`?

Reply:
Done